.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- Thanks for contributing to ArTEMiS! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
+
+### Checklist
+- [ ] I've run `yarn run webpack:build:main` from the root directory to see that the project builds without errors.
+- [ ] I've removed unnecessary whitespace changes.
+- [ ] I've updated the documentation and models if necessary.
+
```

Comment:
Please add a checkmark for I've tested the changes and all related features on the Artemis test server